examples/vyper/default.vy

```diff
@@ -0,0 +1,15 @@
+stored_number: uint256
```

Comment:
We typically only put `square` in here, but this is fine :) Thanks for changing this to work on all the versions; I had to make local changes when I tested with an earlier version on this PR.